index.html

```diff
@@ -39,17 +39,142 @@ <h3 class="connect">LET'S CONNECT</h3>
+        </div>
+      </div>
+
+      <div class="projects">
+        <img src="./images/Snapshoot Portfolio(3).png" alt="">
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img src="./images/Counter.png" alt="Counter"> <span class="dev"> End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <section class="about">
+        <div>
+          <div class="first-heading">
+            <h1>About Myself</h1>
+          </div>
+          <div class="description">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+              work
+              and
+              experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          </div>
+          <div>
+          <h3 class="connect">LET'S CONNECT</h3>
+          <ul class="social">
+            <li class="social"><a href="#"><img src="./images/Vector.svg" alt=""></a></li>
+            <li class="social"><a href="#"><img src="./images/Linkedin icon.svg" alt="Linkedin"></a></li>
+            <li class="social"><a href="#"><img src="./images/Vector (1).svg" alt=""></a></li>
+            <li class="social"><a href="#"><img src="./images/Vector (2).svg" alt="github"></a></li>
+            <li class="social"><a href="#"><img src="./images/Vector (3).svg" alt="twitter"></a></li>
+          </ul>
+        </div>
+          <div>
+            <button class="see-project">Get my resume</Button>
+          </div>
+          <div class="union-div">
+            <h2>Languages</h2>
+            <img src="./images/Union.png" alt="">
+          </div>
+          <div class="about-lang">
+            <img src="./images/Ellipse 1.png" alt="">
+            <a href="http://#">JavaScript</a>
+          </div>
+          <div class="about-lang">
+            <img src="./images/Ellipse 1 (1).png" alt="">
+            <a href="http://#">HTML</a>
+          </div>
+          <div class="about-lang">
+            <img src="./images/Ellipse 1 (2).png" alt="">
+            <a href="http://#">CSS</a>
+          </div>
+          <div class="union-div">
+            <h2>Frameworks</h2>
+            <img class="union2" src="./images/Union2.png" alt="">
+          </div>
+          <div class="union-div">
+            <h2>Skills</h2>
+            <img class="union2 space" src="./images/Union2.png" alt="">
+          </div>
+        </div>

```

Comment:
- [ Required] I think it would be better if the `font-size` and `size` of images in the technologies section are consistent with those in the Figma design. Currently, it appears that the `font-size` and image sizes in this section are slightly bigger compared to the same sections in the Figma design. Please see the screenshot below and rectify this.

| Figma design | Your design |
|-------------------| -----------------|
|![Screenshot from 2023-06-03 02-42-24](https://github.com/MargaretTechworld/Portfolio/assets/16392046/4daba03d-624a-4258-840d-7589b221028e)|![Screenshot from 2023-06-03 02-36-53](https://github.com/MargaretTechworld/Portfolio/assets/16392046/4b756c74-258f-4c2d-9a08-30c6455655ce)|